modules/file-system/parallelstore/variables.tf

```diff
@@ -60,7 +60,11 @@ variable "mount_options" {
 }
 
 variable "private_vpc_connection_peering" {
-  description = "The name of the VPC Network peering connection."
+  description = <<-EOT
+    The name of the VPC Network peering connection.
+    If using new VPC, please use community/modules/network/private-service-access to create private-service-access and
+    If using existing VPC with private-service-access enabled, set this manually."
```

Comment:
I would prefer a link to an example of documentation. If documentation is coming in separate PR then please add a link from here then.

Reply:
documentation should include how to find the name if I am setting it manually.

Reply:
Documentation in #2740 will cover information about how to use existing-vpc if it has private-service-access enabled.